src/FloorSubsystem/FloorSystem.java

```diff
@@ -26,7 +26,7 @@ public FloorSystem(String floorDataFilename) {
 	 */
 	public void sendFloorData() {
 		for (byte[] floorEntry : floorDataEntry) {
```

Comment:
some comments dealing with the whole thing since I can't comment on what isn't edited:
why is floorDataEntry is public and being changed without setters?

Reply:
made it a private static var, used a static setter method since there isn't any point in instantiating FloorSystem in FloorDataParser